src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -214,6 +208,28 @@ protected function writePrompt(OutputInterface $output, Question $question)
         $output->write($message);
     }
 
+    /**
+     * @param string $tag
+     *
+     * @return string[]
+     */
+    protected function prepareChoices($tag, ChoiceQuestion $question)
+    {
+        $choices = $question->getChoices();
+
+        $maxWidth = max(array_map([$this, 'strlen'], array_keys($choices)));
```

Comment:
`static::strlen`?

Reply:
`[static::class, 'strlen']` (but honestly I've no issue with using $this)

Reply:
We currently use `self::` everywhere, we don't expect those methods to be overidden.